docs/reference/sql/endpoints/odbc/configuration.asciidoc

```diff
@@ -260,7 +262,110 @@ image:images/sql/odbc/env_var_log.png[]
 
 NOTE: When enabling the logging through the environment variable, the driver will create *one log file per process*.
 
-Both ways of configuring the logging can coexist and both can use the same destination logging directory. However, one logging message
-will only be logged once, the connection logging taking precedence over the environment variable logging.
+Both ways of configuring the logging can coexist and both can use the same
+destination logging directory. However, one logging message will only be logged
+once, the connection logging taking precedence over the environment variable
+logging.
+
+==== Connection string parameters
+
+The following is a list of additional parameters that can be configured for a
+particular connection, in case the default behavior of the driver is not
+suitable. This can be done within the client application, in a manner
+particular to that application, generally in a free text input box (sometimes
+named "Connection string", "String extras", or similar).  The format of the
+string is `Attribute1=Value1`. Multiple attributes can be specified, separated
+by a semicolon `Attribute1=Value1;Attribute2=Value2;`. The attribute names are
+given below.
+
+===== Timeout
```

Comment:
I suggest using either a table or like in the JDBC chapter a list by using `entry::` syntax: https://github.com/elastic/elasticsearch/blob/47e71b988ee550409ef1863354061f17d298050d/docs/reference/sql/endpoints/jdbc.asciidoc#essential
Also do note the use of links `[[link-name]]` which are useful for linking parts of the docs directly

Reply:
Great point, used entries.
The link seems to only works for a section (not entries), so I've added one for the entire section.